.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python: 2.7
```

Comment:
why? It's only the tests itself that is actual python. All the rest is just grok patterns.